docs/usage_guide.rst

```diff
@@ -44,6 +68,24 @@ If after you have set up premium you wish to replace or disassociate keys with t
    :alt: Delete up Rotki premium API key/secret pair in a premium account
    :align: center
 
+.. _sync_data_with_rotki_server:
+
+Sync data with Rotki Server
+-----------------------------
+
+To back up your data in the Rotki Server switch on "Allow data sync with Rotki Server". This allows you to restore it later on in any account/device as long as :red:`the same API Key/secret and account password are used`.
+
+.. image:: images/rotki_premium_set_sync_data.png
+   :alt: Sync data with Rotki Server
+   :align: center
+
+Bear in mind that in case of using multiple accounts/devices with the data sync enabled, the one with the most recent login will upload the most up-to-date data to Rotki Server. After that, log into another account/device may ask you to take action regarding whether replacing or not your local database with the remote one.
```

Comment:
> After that, log into another account/device may ask you to take action regarding whether replacing or not your local database with the remote one.

After that using the same account from another device, may ask you whether you want to replace your local database with the remote one.